mk/spksrc.service.use_alternate_tmpdir

```diff
@@ -0,0 +1,14 @@
+
+# create and define alternate TMPDIR, can be useful to avoid errors like:
+# Could not install packages due to an EnvironmentError: [Errno 28] No space left on device
+# as /tmp might be too small.
+# typically used to install a huge amount of wheels with pip (see homeassistant package)
+# TMPDIR is supported by pip https://github.com/pypa/pip/issues/4462 ('--build dir' is not supported anymore)
+
+TMPDIR=${SYNOPKG_PKGDEST}/tmp
```

Comment:
I would recommend to add a cleanup process to discard this "tmp" folder after installation, typically in "post_install" function.

Reply:
✔️

Reply:
@publicarray as in DSM7 the `target/tmp` folder is provided by the system, this $TMPDIR handling might need a rework for dsm7.